src/explore-education-statistics-admin/src/pages/release/pre-release/ReleasePreReleaseAccessPage.tsx

```diff
@@ -39,55 +39,53 @@ const ReleasePreReleaseAccessPage = () => {
     <LoadingSpinner loading={isLoading}>
       {release && (
         <Tabs id="preReleaseAccess">
-          {!release.amendment && (
```

Comment:
I bet the realisation that removing this was all that was necessary was a sweet one.